lib/litle.rb

```diff
@@ -27,6 +27,14 @@
 module ActiveMerchant #:nodoc:
   module Billing #:nodoc:
     class LitleGateway < Gateway
+      def initialize(options={})
+        if options.has_key?(:test_url)
```

Comment:
I made the change to test with prelive environment. I think setting `test_url` in the yml was ignored until then. It's a [class attribute ](https://github.com/activemerchant/active_merchant/blob/master/lib/active_merchant/billing/gateway.rb#L134 )in activemerchant not an option